config/odp-linux-generic.conf

```diff
@@ -33,6 +33,10 @@ shm: {
 	# When using process mode threads, this value should be set to 0
 	# because the current implementation won't work properly otherwise.
 	num_cached_hp = 0
+
+	# Allocate internal shared memory using a single virtual address space.
+	# Set to 1 to enable using process mode.
+	single_va = 0
```

Comment:
Need to add a validation test that sets `single_va = 1` so that we know this actually works. Is this something planned for a follow-on PR?

Reply:
Yes, process mode is not yet working but I'll add a test when I have all the fixes done.